RULES.md

```diff
@@ -285,6 +286,38 @@ model User {
 }
 ```
 
+#### With `{ style: "singular", allowlist: ["UserData"] }`
+
+```prisma
+// good
+model UserData {
+  id String @id
+}
+
+// bad
```

Comment:
It isn't immediately obvious to me why TenantData is considered bad in this example.

Is it because `Data` is considered plural by pluralize library? Or because it is not specified in `allowlist`?

Reply:
Thanks for calling this out. I added more example and a comment to clarify that data is considered plural by default.